composer.json

```
@@ -20,8 +20,10 @@
     },
     "require": {
         "php": "^7.1",
-        "symfony/config": "^3.0",
-        "symfony/dependency-injection": "^3.0",
-        "symfony/http-kernel": "^3.0"
+        "symfony/config": "^3.4",
```

I think we can support both 3.4 and 4+
```suggestion
        "symfony/config": "^3.4||^4.0",
```

---

Are you sure? I don't have sure yet if we are ensuring the compatibility with Symfony 4 and its variants.

---

Yes, it is.

---

Done! :+1: